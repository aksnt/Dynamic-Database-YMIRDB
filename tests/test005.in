SET a 1 2 3
SET b 4 5 6
LIST SNAPSHOTS
SNAPSHOT
SET b 7 8 9
SET a 10 10 b
SNAPSHOT
CHECKOUT
POP A
POP A
POP A
POP A
GET A
GET B
