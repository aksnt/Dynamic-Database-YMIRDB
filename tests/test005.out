> ok

> ok

> no snapshots

> saved as snapshot 1

> ok

> ok

> saved as snapshot 2

> ok

> 1

> 2

> 3

> nil

> []

> [4 5 6]
